sdk/formrecognizer/ai-form-recognizer/src/formRecognizerClient.ts

```diff
@@ -80,9 +82,28 @@ export type BeginRecognizeContentOptions = RecognizeContentOptions & {
    */
   resumeFrom?: string;
   /**
-   * Content type of the input. Supported types are "application/pdf", "image/jpeg", "image/png", and "image/tiff".
+   * Content type of the input. Supported types are "application/pdf", "image/jpeg", "image/png", "image/tiff", and "image/bmp".
    */
   contentType?: FormContentType;
+  /**
+   * The BCP-47 language code of the text in the document.
+   *
+   * The supported languages are English ('en'), Dutch (‘nl’), French (‘fr’), German (‘de’), Italian (‘it’), Portuguese (‘pt'),
```

Comment:
There are docs that show supported languages if you want to use them: https://docs.microsoft.com/azure/cognitive-services/form-recognizer/language-support